src/index.js

```diff
@@ -45,7 +45,7 @@ export default class Autocomplete extends Component {
 		onDelete: PropTypes.func,
 		onFocus: PropTypes.func,
 		onBlur: PropTypes.func,
-		onChange: PropTypes.func,
+		placeholder: PropTypes.string,
```

Comment:
We need to remove `onChange: PropTypes.func`?

Reply:
My bad :smile: fixed!